substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/xml/JavaxXmlClassAndResourcesLoaderFeature.java

```diff
@@ -36,13 +36,22 @@
 
 import com.oracle.svm.core.feature.InternalFeature;
 import com.oracle.svm.core.jdk.JNIRegistrationUtil;
+import com.oracle.svm.util.ReflectionUtil;
 import com.oracle.svm.core.feature.AutomaticallyRegisteredFeature;
 
 @AutomaticallyRegisteredFeature
 public class JavaxXmlClassAndResourcesLoaderFeature extends JNIRegistrationUtil implements InternalFeature {
 
     @Override
     public void beforeAnalysis(BeforeAnalysisAccess access) {
+        // Ensure that class loading of xml related classes work
+        // com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl
+        // implicitly depends on jdk.xml.internal.JdkXmlUtils. So initialization
+        // order needs to happen in the reverse order.
```

Comment:
General remark: for multi-line text, we prefer block comments (`/* */`) and full sentences with terminal punctuation, because that plays better with the formatter wrt rearranging line breaks. No blocker, just a hint for the future. :)

Reply:
OK, I'll fix.

Reply:
Should be fixed now.